spack.yaml

```diff
@@ -19,9 +19,9 @@ spack:
       require:
         - '@git.dev_2024.08.14=access-esm1.6'
         - '+access-gtracers'
-    cice4:
+    cice5:
       require:
-        - '@git.2024.05.21=access-esm1.5'
+        - '@git.access-esm1.6=access-esm1.6'
```

Comment:
The cice5 SPR doesn't contain version `access-esm1.6` yet.

Reply:
Yeah agreed. But that should cause the build to fail, rather than what happened here, which spack solved and installed cice4.

Reply:
Good point! We need to work how how that happened.

Reply:
Good point! We need to work out how that happened.